src/app/routes/index.test.jsx

```diff
@@ -19,15 +19,16 @@ import sportArticlePageJson from '#data/sport/judo/articles/cj80n66ddnko.json';
 import mediaAssetPageJson from '#data/yoruba/cpsAssets/media-23256797.json';
 
 import { ERROR_PAGE, FRONT_PAGE } from '#app/routes/utils/pageTypes';
+import * as fetchDataFromBFF from '#app/routes/utils/fetchDataFromBFF';
 import gahuzaOnDemandAudio from '#data/gahuza/bbc_gahuza_radio/p02pcb5c.json';
+// eslint-disable-next-line import/order
```

Comment:
Can we fix the import order rather than disabling the lint rule?

Reply:
I tried this in an [earlier commit](https://github.com/bbc/simorgh/pull/12224/commits/e10ee36141a2ebe5cc913afd029de8e0bf0aeb76), but github tests failed. Similar solutions of [rearranging imports worked in other files](https://github.com/bbc/simorgh/pull/12224/commits/4110e8089891a42a6f1db9d9dba703d07fe4b616). But this file doesn’t like it

Reply:
Apologies, I didn't see your comment when I added this review. I'm OK with this - it's only a test file, and I'm pleased to see that the other imports are now using aliases (as it means that if we copy this file / look at it for inspiration, we would follow the same patterns).

Reply:
All good. and again we don’t have to include these specific import lines to complete the work described by the Jira ticket. There are other import statements that still use relative pathing aswell, but these import statements also weren’t flagged by linting checks for `eslint-config-airbnb 19.0.4.`

Reply:
That's interesting - I wonder why they are not included in the linting checks. It's not something to worry about for now - we just want to do the bare minimum to allow us to upgrade the eslint-config-airbnb library. Thanks for the update!